packages/cli/src/ui/contexts/KeypressContext.tsx

```diff
@@ -374,7 +374,9 @@ function* emitKeys(
             ch = yield;
             sequence += ch;
           }
-        } else if (ch === '<') {
+        }
+
+        if (ch === '<') {
```

Comment:
this was previously in an else. should we be considering sequences that had a `;` SGR mouse mode. If this is legit matching more real SGR mouse sequences great but would be worried this might false positive.